lib/rubyblok/helpers/storyblok_helper.rb

```diff
@@ -35,13 +36,28 @@ def rubyblok_blocks_tag(bloks)
     render_inline_partial template:, locals: { bloks: }
   end
 
+  def rubyblok_link_tag(text, content)
```

Comment:
the input parameter should be the link option hash provided by storyblok, it is something like
```
"link"=>{"id"=>"", "url"=>"[gmail.com](http://gmail.com/)", "linktype"=>"url", "fieldtype"=>"multilink", "cached_url"=>"[gmail.com](http://gmail.com/)"},
```

Reply:
I need to specify a text on the method, so I keep like this `rubyblok_link_tag(text, options)` also is using the `cached_url`